src/test/java/emissary/util/PayloadUtilTest.java

```diff
@@ -172,6 +172,7 @@ public void testIsValidForm() {
         assertTrue("Dash and underscore aren't considered valid", PayloadUtil.isValidForm("-_"));
         assertTrue("Parentheses aren't considered valid", PayloadUtil.isValidForm("formName-(suffixInParens)"));
         assertTrue("Slash aren't considered valid", PayloadUtil.isValidForm("formName-(application/xml)"));
+        assertTrue("Plus aren't considered valid", PayloadUtil.isValidForm("formName-(application+xml)"));
         assertFalse("Dot isn't considered valid", PayloadUtil.isValidForm("."));
```

Comment:
LOL - given that the initial driver for this validation was catching forms that contain a embedded period, it would be nice for our "dot" testing to use a more representative form than `"."`.